tests/php/Security/MemberTest.php

```diff
@@ -1618,4 +1621,111 @@ public function testChangePasswordToBlankIsValidated()
         $result = $member->changePassword('');
         $this->assertFalse($result->isValid());
     }
+
+    public function testMapInCMSGroupsNoLeftAndMain()
+    {
+        if (class_exists(LeftAndMain::class)) {
```

Comment:
LeftAndMain will always exist in practice, as silverstripe/admin is installed as part of silverstripe/installer in CI

I guess this is still a required test due to the logic in Member::mapInCMSGroups() .. though :/

Reply:
Yeah it won't run the way our CI is currently set up - there are likely other tests like that too. I'd like to keep it and if we make upgrades to our CI in the future to let framework run without installer then it'll be run.